theme/padplus/scss/post/base/_fonts.scss

```diff
@@ -0,0 +1,43 @@
+body {
+    font-family: Roboto !default;
```

Comment:
!default is a sass flag which makes this rule ignored, since font-family has already been defined in the Boost theme. If we need to change fonts, we can always use $font-family-base in pre config. But Roboto is not installed on all systems anyway...